packages/appkit/src/utils/TypesUtil.ts

```diff
@@ -100,7 +100,7 @@ export type AppKitOptions = {
   universalProvider?: UniversalProvider
   /**
    * The default account type used for each chain namespace.
-   * @default "{ bip122: 'payment', eip155: 'smartAccount', polkadot: 'eoa', solana: 'eoa' }"
+   * @default "{ bip122: 'payment', eip155: 'smartAccount', polkadot: 'eoa', solana: 'eoa', ton: 'eoa' }"
```

Comment:
this is funny cause they are actually SAs but there's no meaning/difference in TON